lom/src/plugins/sonic/client/dbclient/counter_repository.go

```diff
@@ -46,33 +56,58 @@ func (counterRepository *CounterRepository) GetInterfaceCounters() (InterfaceCou
         }
 
     for interfaceName, interfaceOid := range interfaceToOidMapping {
-
-        interfaceCountersKey := counters_db_table_name + interfaceOid
-        fields := []string{sai_port_stat_if_in_errors_field, sai_port_stat_if_in_ucast_pkts_field, sai_port_stat_if_out_ucast_pkts_field}
-        result, err := counterRepository.RedisProvider.HmGet(COUNTER_DB_ID, interfaceCountersKey, fields)
-
+        isInterfaceActive, err := counterRepository.IsInterfaceActive(interfaceName)
```

Comment:
As discussed. Add a TODO for getting active i/f from a different Go routine that subscribes to AppDB, where updates are seldom.

Reply:
added todo.